recipes/namd/3.0/mc/extra/reframe.yaml

```
@@ -0,0 +1,10 @@
+develop:
+  features:
+    - namd
```

does the `develop` view also provide `namd`?
should this be `namd-build` (or a similar name that indicates that namd can be built)?

---

Well spotted, thanks! The `develop` view does not provide `namd`. In the `gh200` packaged I used the `-dev` suffix.
```suggestion
    - namd-dev
```